draft-ietf-moq-requirements.md

```diff
@@ -432,6 +432,10 @@ Our goal in this section is to understand the requirements that result from the
 
 In order to support the various topologies and patterns of media flows with the protocol, the protocol MUST support both sending and receiving of media streams, as separate actions or concurrently in a given connection.
 
+### QUIC Capabilities and Properties
+
+With QUIC being the underlying protocol brings capabilities and functionalities for many of the requirements such as connection migration and re-use, greater controls over packet reliability, congestion control, re-ordering and flow directionality, multiplexing and head of line blocking. Utilising aspects of the QUIC protocol which would then necessitate reimplementation of such capabilities already present in other parts of the protocol should only be done so if requirements deem them incompatible.
```

Comment:
So, three things here: 

Thanks for adopting the one-line-per-sentence convention. I wish I'd thought of that years ago - it really does help with merge conflicts. 

I'm not sure **which** protocol this phrase is referring to - I can guess, but it is a guess - 

> which would then necessitate reimplementation of such capabilities already present in other parts of the protocol

and that realization led to another realization - most of the BOF and working group discussion talked about a "MOQ protocol", but we now seem to have a MOQ **transport** protocol. 

Is this referring to "MOQT" (if so, inserting a reference to MOQT would be good - our references date to pre-BOF individual drafts, and are only used in the ACKS section), or to a MOQ system?

Reply:
Quite honestly, it's not referring to a specific (existing) protocol but talking in much more general terms. In reality it's more applicable for media encapsulation protocols (e.g. RUSH/WARP) than MOQT. Should I be more specific here?